README.md

```diff
@@ -207,6 +207,21 @@ For sync communication we have a community slack with a #containerd channel that
 
 __If you are reporting a security issue, please reach out discreetly at security@containerd.io__.
 
+## Applying License Header to New Files
+
+If you submit a contribution that adds a new file, please add the license
+header. You can do so manually or use the `ltag` tool:
+
+
+```console
+$ go get github.com/kunalkushwaha/ltag
+$ ltag -t ./script/validate/template
+```
+
+The above will add the appropriate licenses to Go, Makefile, Dockerfile files & Bash-scripts. New templates will
```

Comment:
maybe this can be re-worded as follows to make it more readable:

...*add the appropriate license header to Go language source files, Makefiles, Dockerfiles, and shell scripts.*